### DISCUSSION TITLE
Enhancement - Ignore .github Folder

### DISCUSSION THREAD

**Discussion by CollinHerber** [mkdocs/mkdocs#3792] [Q&A] [answered]
Something I'd like to setup is Github Actions to run when docs are deployed.
The gh-deploy will delete the .github folder including the workflows so it makes adding CI/CD on top of this impossible.

For example I am not using GH pages for my docs, but rather an external tool to host it so I'd like my files to be deployed and then CI after that deploys the folder.

I tried utilizing something like `mkdocs build` and just using that but it doens't appear to contain everything it needs (the top level index.html is missing for some reason with that?)

**Comment by pawamoy** [comment, marked as answer]
Hi @CollinHerber, thanks for the enhancement suggestion. I think the way forward for you is to indeed stop using the `gh-deploy` command and instead run Git commands manually (or with a custom script).

The `gh-deploy` command is really just a shortcut to build the site, commit it to the gh-pahes branch (at the root of the repo), and push it. I suggest you do something similar manually, which will allow you to keep any extra files suc as your GHA workflow, or maybe even change the file layout to build your site in a subfolder instead of the root of the repository.

I don't think adding a special case to it, to somehow prevent deleting `.github` is a good idea.
